nav2_bringup/launch/tb3_simulation_launch.py

```diff
@@ -30,6 +30,7 @@ def generate_launch_description():
     # Get the launch directory
```

Comment:
What position does this default to spawning in? Is it a good position? This is the first look everyone will have for Nav2 when launching, so it would be good if this was a sensible location (on one of the aisle ways, maybe somewhere central like where I spawned in the readme here: https://github.com/ros-planning/navigation2/tree/main/nav2_simple_commander

Reply:
![Screenshot from 2021-12-06 11-36-50](https://user-images.githubusercontent.com/24660508/144831607-4b711e02-d519-477c-8f1b-f319f04c3d1c.png)

Could it work?

Reply:
I'd prefer in that green "T" aisleway you see above. Some of the examples are keepout zones, which those yellow boxes are clearly going to be part of. Don't want to spawn the robot in the middle of a keepout / danger zone :laughing:

Reply:
![Screenshot from 2021-12-07 22-54-03](https://user-images.githubusercontent.com/24660508/145111980-a44fbf76-716b-4dae-aeb1-87b0729b560c.png)
 
Thrown position changed.